build

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
On pretty much any Linux system you'll find bash at /bin/bash. If you want to be 100% we're running the first bash found in the user's $PATH, rather than making an assumption of the path, then this might be better:

```#!/usr/bin/env bash```